src/App.tsx

```diff
@@ -0,0 +1,59 @@
+import useSWRImmutable from "swr/immutable";
+import { get } from "./api/api";
+import { meroppfolgingApiUrl } from "./api/urls";
+import { Fetcher } from "swr";
+import React from "react";
+import { SenOppfolgingStatusDTO } from "./schema/senOppfolgingStatusSchema";
+import { MikrofrontendLinkPanel } from "./components/panels/MikrofrontendLinkPanel";
+
+function App() {
+  const fetchSenOppfolgingStatus: Fetcher<SenOppfolgingStatusDTO, string> = (path) => get(path);
+  const { data, error } = useSWRImmutable(meroppfolgingApiUrl, fetchSenOppfolgingStatus);
+
+  if (error) throw error;
+
+  if (data && data.isPilot === true && data.responseStatus == "NO_RESPONSE") {
+    const maxDateText =
+      data.maxDate != null
+        ? `Den ${data.maxDate} er datoen du ikke lenger vil motta sykepenger (maksdato).`
+        : `Det nærmer seg datoen du ikke lenger vil motta sykepenger.`;
+
+    return (
+      <MikrofrontendLinkPanel
+        headingText="Snart slutt på sykepengene"
+        bodyText={`${maxDateText} Vi trenger derfor at du svarer på to spørsmål om situasjonen du står i så vi kan hjelpe deg.`}
```

Comment:
Litt usikker på om det er lurt å skrive "to spørsmål" da vi muligens kommer til å åpne for flere spørsmål på sikt. Det er ikke verdens undergang, men da må man huske på å holde tekst i mikrofrontend i synk med antall spørsmål i frontend. Kan evnt vurdere å bytte ut med "noen spørsmål" eller noe sånt

Reply:
Ja, jeg lurer på om det er en måte å fortelle brukeren at det ikke er mye å svare på. Kanskje skrive "noen få spørsmål" i stedet?

Reply:
Høres bra ut :)